tests/ops/op_math/test_linear_combination.py

```diff
@@ -53,24 +53,7 @@ def test_isinstance_Hamiltonian(self):
         assert isinstance(H, qml.Hamiltonian)
 
 
-@pytest.mark.usefixtures("new_opmath_only")
-def test_mixed_legacy_warning_Hamiltonian():
-    """Test that mixing legacy ops and LinearCombination.compare raises a warning"""
-    op1 = qml.ops.LinearCombination([0.5, 0.5], [X(0) @ X(1), qml.Hadamard(0)])
-
-    with pytest.warns(
-        qml.PennyLaneDeprecationWarning,
-        match="Using 'qml.ops.Hamiltonian' with new operator arithmetic is deprecated",
-    ):
-        op2 = qml.ops.Hamiltonian([0.5, 0.5], [qml.operation.Tensor(X(0), X(1)), qml.Hadamard(0)])
-
-    with pytest.warns(UserWarning, match="Attempting to compare a legacy operator class instance"):
-        res = op1.compare(op2)
-
-    assert res
-
-
-@pytest.mark.usefixtures("legacy_opmath_only")
+@pytest.mark.usefixtures("use_legacy_and_new_opmath")
```

Comment:
Actually the `use_legacy_and_new_opmath` fixture can be removed because everything is going to be tested with both legacy and new opmath unless marked `new_opmath_only` or `legacy_opmath_only`. The `use_new_opmath` fixture can also be removed because it doesn't really do anything.